front/notificationmailingsetting.form.php

```diff
@@ -51,6 +52,10 @@
     }
     $config = new Config();
     $config->update($_POST);
+    Event::log(0, "system", 3, "setup", sprintf(
+        __('%1$s edited the emails notifications configuration'),
+        $_SESSION["glpiname"] ?? __("Unknown"),
+    ));
```

Comment:
```suggestion
    if ($config->update($_POST)) {
        Event::log(0, "system", 3, "setup", sprintf(
            __('%1$s edited the emails notifications configuration'),
            $_SESSION["glpiname"] ?? __("Unknown"),
        ));
    }
```

Reply:
I tried this at first but realized that it did not work because `$config->update($_POST)` actually return false...

Reply:
indeed